app/models/referentiels/api_referentiel.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class Referentiels::APIReferentiel < Referentiel
+  validates :mode, inclusion: { in: ['exact_match', 'autocomplete'] }, allow_blank: true, allow_nil: true
+
+  before_save :name_as_url
```

Comment:
pourquoi ? c'est juste temporaire pour satisfaire la contrainte sur name not null ?

Reply:
oui, c'est une "contrainte" du csv, et a l'avoir rencontré, je me dis que c'est pas si mal si(/quand) on souhaite(/ra) rendre "explorable" les referentiels existant